ironic_prometheus_exporter/parsers/ipmi.py

```diff
@@ -71,17 +79,28 @@ def extract_labels(entries, payload, node_name):
     LOG.info('extract_labels function called with: entries=%s | payload=%s | \
              node_name=%s' % (str(entries), str(payload), node_name))
     if len(entries) == 1:
+        status = payload[entries[0]].get('Status')
         labels = {'node_name': node_name,
```

Comment:
I'm concerned that calling this "node_name" is going to be confusing in a kubernetes environment, where k8s nodes and ironic nodes are very different things and won't necessarily have the same names.

Reply:
@dhellmann I just used the field that comes in the data,  should I just remove or should we give another name?